polymorphia-backend/src/main/java/com/agh/polymorphia_backend/controller/CourseGroupsController.java

```diff
@@ -19,27 +20,61 @@
 public class CourseGroupsController {
     private final CourseGroupsService courseGroupService;
 
-    @GetMapping(value = "/all")
+    @PostMapping()
+    @PreAuthorize("hasAnyAuthority('COORDINATOR')")
+    public ResponseEntity<Void> createCourseGroup(@RequestBody CreateCourseGroupRequestDto requestDto) {
+        courseGroupService.createCourseGroup(requestDto);
+        return ResponseEntity.ok().build();
```

Comment:
```suggestion
        return ResponseEntity.noContent().build();
```

applies to other places where appropriate